protocol/src/main/java/org/jboss/as/protocol/ProtocolConnectionUtils.java

```diff
@@ -163,11 +167,36 @@ public static Connection connectSync(final ProtocolConnectionConfiguration confi
         final AuthenticationContext context = AuthenticationContext.empty().with(MatchRule.ALL, mergedConfiguration);
 
         if (clientBindAddress == null) {
-            return endpoint.connect(uri, options, context);
+            try {
+                return AccessController.doPrivileged((PrivilegedExceptionAction<IoFuture<Connection>>) () ->
+                        endpoint.connect(uri, options, context));
+            } catch (PrivilegedActionException pae) {
+                ProtocolLogger.CONNECTION_LOGGER.connectionPrivilegedAccessError(uri, pae);
```

Comment:
We don't normally both log *and* rethrow exceptions as that adds undue noise when the exception bubbles up again.

The PAE itself does not add useful information so it's better to just discard it completely and only throw the cause.